### PR TITLE
Add WebdriverIO application

### PR DESCRIPTION
**Pull request by christian-bromann**
Please let me know if any points are not clear or should be extended.

Thanks!

### CODE REVIEW

**Comment by kborchers** [comment]
Hello @christian-bromann! Thank you for the application. I will make sure the TAC is aware and begins reviewing ASAP. We will be in touch soon.

**Comment by dmethvin** [comment]
Hey @christian-bromann, the TAC is reviewing this now and will let you know soon!

File: Applications/webdriverio.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Any available metrics or even estimates about the user base, ecosystem and community.
+
+* NPM stats:
+    * ~18k daily downloads (https://npm-stat.com/charts.html?package=webdriverio)
+    * 148 dependent projects
+    * Automation driver for tools like Spectron (official Electron Testing Framework maintained by GitHub), Chimp and CodeceptJS
+* Gitter support Channel: ~1800 user (https://gitter.im/webdriverio/webdriverio)
+* Most active authors
+
+![Active users](https://mo.github.io/assets/javascript-e2e-integration-testing-active-devs.png)
```
Comment (dmethvin) [inline Applications/webdriverio.md]: Do you have a definition for "active authors"? The graph seems to indicate there were about 150 active authors in the past year.

File: Applications/webdriverio.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+# Project scope.
+
+The goal of WebdriverIO is to provide developers a framework to run e2e tests on their desktop and mobile applications. It tries to simplify the test process by taking care of common pitfalls in e2e testing. It is not made to run any kind of other tests like unit or API tests. It only speaks the Webdriver protocol. However services and add ons may provide additional functionalities.
+
+# Current governance process.
```
Comment (yichoi) [inline Applications/webdriverio.md]: It would be better to add detailed government process to be.

File: Applications/webdriverio.md (hunk 1 above)
Comment (christian-bromann) [reply Applications/webdriverio.md]: @dmethvin with active authors I meant people who contributed in the past year. Obviously some of them are one time contributors however some also proposed PRs multiple times.

File: Applications/webdriverio.md (hunk 2 above)
Comment (christian-bromann) [reply Applications/webdriverio.md]: @yichoi what kind of details are you missing? I'ld be happy to provide more info.

File: Applications/webdriverio.md (hunk 1 above)
Comment (dmethvin) [reply Applications/webdriverio.md]: @christian-bromann How/where was that data collected regarding contributors?  I don't see 150 contributors in the graphs that GitHub provides but maybe your data crosses multiple repos, counts issue creators, etc.

File: Applications/webdriverio.md (hunk 2 above)
Comment (kborchers) [reply Applications/webdriverio.md]: @christian-bromann If you wanted to add some sort of intended governance process here that would be great but we had an internal discussion and clarified that if that is unknown at the time and all you know is what you do now, that is fine. The mentorship process is meant to develop and implement that plan so if you don't have anything to add here, that is ok.

File: Applications/webdriverio.md (hunk 1 above)
Comment (christian-bromann) [reply Applications/webdriverio.md]: @dmethvin it was generated using git source metrics by someone who did a full e2e tooling  comparison on his blog: https://mo.github.io/2017/07/20/javascript-e2e-integration-testing.html. I grabbed the stats from there.

Comment (christian-bromann) [reply Applications/webdriverio.md]: The source of graph generation is https://github.com/mo/git-source-metrics-web-e2e

File: Applications/webdriverio.md (hunk 2 above)
Comment (christian-bromann) [reply Applications/webdriverio.md]: > if that is unknown at the time and all you know is what you do now, that is fine.

This is the case. There is no defined governance process yet. I would be happy to develop one as part of the JSF mentorship

File: Applications/webdriverio.md (hunk 1 above)
Comment (dmethvin) [reply Applications/webdriverio.md]: When I looked at the repo it didn't look that high but i just pulled the repo and did a `$ git shortlog 9125db7..HEAD` which came pretty close to the same number. So we're good :+1:

**Comment by dmethvin** [comment]
@christian-bromann Congratulations! 🎉 Can you send an email to projects@js.foundation for next steps? Thanks!

**Pull request closed without merging**